scc/install-ootb-sc-basic.md

```diff
@@ -113,6 +119,11 @@ To install Out of the Box Supply Chain Basic:
       ssh_secret: git-ssh
       commit_strategy: direct
 
+    maven:
+      repository:
+        url: https://my-maven-repository/releases
```

Comment:
i.e. team foo might use Nexus, team bar uses Artifactory, team cheese uses GitHub packages

Reply:
Since Workload params override ClusterSupplyChain params, yes. Generally, we expect the Application Operator to define the maven repository url and provide credentials to connect to it. The Workload is primarily intended for the Application Developer.

Reply:
^ this is a theory we need to validate and answer before we add the content of the `maven` param as  Workload `.spec.source.maven`